lib/r10k/util/subprocess/runner/jruby.rb

```diff
@@ -14,8 +14,7 @@ def initialize(argv)
   end
 
   def run
-    spawn_opts = @cwd ? {:chdir => @cwd} : {}
-    stdout, stderr, status = Open3.capture3(*@argv, spawn_opts)
+    stdout, stderr, status = Open3.capture3(@argv.join(' '))
```

Comment:
Well, the `cwd` value is currently only used by the SVN code, so if this is just JRuby then this is less of a worry.